src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -877,52 +973,52 @@ public void setGerritVerifiedCmdBuildNotBuilt(String cmd) {
 
     @Override
     public Integer getGerritBuildStartedVerifiedValue() {
```

Comment:
These getters should perhaps be deprecated?

Reply:
Done